### ISSUE TITLE
LMDE4 wont boot live installer Ryzen 3400G in UEFI Mode

### ISSUE THREAD

**Issue by frostyturnip** [linuxmint/lmde-4-cinnamon-beta#50]
ASrock X470 Mini-ITX motherboard and 3400G processor. Booting to live USB in CMS mode works, but in in UEFI mode even in compatibility mode it fails after the logo showing. Black screen with blinking cursor. Responds to REISUO. Same live USB installer works perfectly with Intel i-5 8500 system on ASrock 390Z Mini-ITX in UEFI mode.

**Comment by paulvonlux** [comment]
i don't think it has the latest kernel... it's the Debian main  4.19.x kernel... 5.x kernel should do the trick but that is LMDE 5

**Comment by frostyturnip** [comment]
3400G system  boots to the installer LMDE 3 albeit in software rendering mode. So something is missing in LMDE 4 Beta.

**Comment by Dyras** [comment]
@paulvonlux Eh? I thought LMDE 4 had UEFI support? It's based on Debian Buster.
https://www.debian.org/releases/buster/amd64/release-notes/ch-whats-new.en.html#secure-boot

**Comment by frostyturnip** [comment]
> **paulvonlux**

It does,  as I stated with i5-8500 system.

**Comment by paulvonlux** [comment]
> 
> 
> > **paulvonlux**
> 
> It does, as I stated with i5-8500 system.

it does not have kernel 5.x... i use latest kernel 5.5.8 , updated with a special script..
